assets/src/blocks/Covers/TakeActionCovers.js

```diff
@@ -36,30 +36,39 @@ export const TakeActionCovers = ({ initialRowsLimit, covers, row, loadMoreCovers
           }
 
           return (
-            <div key={title} className='col-lg-4 col-md-6 cover-card-column'>
-              <div className='cover-card card-one' style={{ backgroundImage: `url(${image})` }}>
+            <div key={title} className='col-lg-4 col-md-6'>
+              <div className='cover-card-new'>
                 <a
                   className='cover-card-overlay'
                   data-ga-category='Take Action Covers'
-                  data-ga-action='Image'
+                  data-ga-action='Card'
                   data-ga-label='n/a'
                   href={button_link}
                   aria-label={__('Take action cover, link to ' + title, 'planet4-blocks')}
                 />
+                <a
```

Comment:
After discussing with Julia we decided to separate click events on the card from the ones on the image, since now the image is no longer the background and therefore not full height anymore